packages/@aws-cdk/aws-ecs/lib/base/task-definition.ts

```diff
@@ -273,6 +272,28 @@ export class TaskDefinition extends cdk.Construct {
     this.placementConstraints.push(pc);
   }
 
+  /**
+   * Extend this TaskDefinition with the given extension
+   *
+   * Extension can be used to apply a packaged modification to
+   * a task definition.
+   */
+  public addExtension(extension: ITaskDefinitionExtension) {
+    extension.extend(this);
+  }
+
+  /**
+   * Create the execution role if it doesn't exist
+   */
+  public obtainExecutionRole(): iam.IRole {
```

Comment:
Does this need to be public?

Reply:
It needs to be accessible from `SomeImage.bind()`. So yes.